modules/twinklewarn.js

```diff
@@ -926,6 +926,10 @@ Twinkle.warn.messages = {
 			label: 'Removing proper sources containing dead links',
 			summary: 'Notice: Removing proper sources containing dead links'
 		},
+		'uw-displaytitle': {
+			label: 'Incorrect use of DISPLAYTITLE',
+			summary: 'Notice: Incorrect use of DISPLAYTITLE'
```

Comment:
Not sure this conveys the intent of the template fully.  It looks like you wrote it to be used for a new account using `DISPLAYTITLE` instead of moving a page; these don't seem to fully capture that, they seem broader to me.  And might be a bit BITEy?

Reply:
I see your point, but I'm not sure how to make it better without making it really really long. Any ideas?

Reply:
Yeah I was struggling too. Maybe something more straightforward, like "DISPLAYTITLE doesn't modify the page name"?  Out of style but as far I've gotten atm.

Reply:
Fine enough I suppose!  Merged.